app/components/lg-lion-search.js

```diff
@@ -39,24 +43,31 @@ export default Ember.Component.extend({
     var params = {},
         selectedGender = this.get('selectedGender'),
         defaultGender = this.get('defaultGender'),
-        selectedOrganization = this.get('selectedOrganization'),
-        selectedName = this.get('selectedName');
+        selectedOrganization = this.get('selectedOrganization.id`'),
```

Comment:
also if we are going to fetch the Id let's called the variable selectedOrganizationId